exporter/splunkhecexporter/go.mod

```diff
@@ -135,3 +135,7 @@ replace github.com/open-telemetry/opentelemetry-collector-contrib/pkg/pdatautil
 replace github.com/open-telemetry/opentelemetry-collector-contrib/pkg/pdatatest => ../../pkg/pdatatest
 
 replace github.com/open-telemetry/opentelemetry-collector-contrib/pkg/golden => ../../pkg/golden
+
+replace go.opentelemetry.io/collector/config/configcompression => github.com/rnishtala-sumo/opentelemetry-collector/config/configcompression v0.0.0-20241118202008-518d136b719c
```

Comment:
@dmitryax -  as requested, I am using core changes from my fork in contrib temporarily.

Reply:
it isn't clear why I still get these errors, don't see them on my local.

`../../exporter/splunkhecexporter/client.go:660:54: undefined: configcompression.TypeWithLevel`